package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "react": "^16.5.0",
     "react-dom": "^16.5.0",
-    "react-scripts": "1.1.5"
+    "react-scripts": "^2.0.0-next.a671462c"
```

Comment:
I am not a huge fan of using `next` versions as they tend to make upgrading painful. In recent times, I had real issues when I used material-ui@next for some of my projects.